File: extra05/keith/ex3/ex3.cpp
#include <iostream>
using namespace std;

int foo(int x, int y) {
  int res = x * y;
  return res;
}

int bar(string s) {
  int n = s.find(":");
  return foo(s.substr(0,n),s.substr(n+1,s.length()));
}

int main() {
  cout << "Testing foo ... " << foo(6,7) << endl;
  cout << "Testing bar ... " << bar("6:7") << endl;
}
  

File: extra05/keith/ex3/Ex3.java
public class Ex3 {
  public static int foo(int x, int y) {
    int res = x * y;
    return res;
  }

  public static int bar(string s) {
    String[] A = s.split(":");
    return foo(A[0],A[1]);
  }

  public static void main(String[] args) {
    System.out.println(
        "Testing foo ... " + foo(6,7));
    System.out.println(
        "Testing bar ... " + bar("6:7"));
  }
}

  
